### DISCUSSION TITLE
Отключение вызова в мобильное приложение

### DISCUSSION THREAD

**Discussion by kulakoff** [rosteleset/SmartYard-Server#41] [Ideas]
У пользователя несколько доступных адресов. При отключении  вызова в мобильное приложение через Admin WEB GUI  вызовы отключаются для всех  адресов.  

![image](https://github.com/rosteleset/SmartYard-Server/assets/1862687/7dbfa769-b8f1-42c6-876a-d9769af8114f)

**Comment by mmikel** [comment]
в админ панели это нужно в последнюю очередь

**Comment by mmikel** [comment]
перенести в дискуссии
